bigtable/docs/snippets.py

```diff
@@ -88,6 +90,8 @@ def setup_module():
     operation = Config.INSTANCE.create(clusters=[cluster])
     # We want to make sure the operation completes.
     operation.result(timeout=100)
+    Config.TABLE = Config.INSTANCE.table(TABLE_ID)
+    Config.TABLE.create()
```

Comment:
There isn't any point in creating this table:  no snippet actually uses it.

Reply:
This is not new code but been moved from list_tables snnipet, so list_tables snippet will be using it.
This is similar to snippets_table.py, where we create common table to be used in snippets where we require table snippets.
If you think this is not required here then we can move back to list_tables, but what I can see advantage of this is we do not need to explicitly delete this table, as it would be gone when instance is deleted in teardown.